test/Passes/dps-in-tile-and-fuse.mlir

```diff
@@ -0,0 +1,80 @@
+// RUN: tpp-opt %s -tile-consumer-and-fuse-producers -cse | FileCheck %s
+
+#map = affine_map<(d0, d1, d2, d3, d4, d5, d6) -> (d0, d2, d4, d6)>
+#map1 = affine_map<(d0, d1, d2, d3, d4, d5, d6) -> (d1, d2, d6 floordiv 2, d5, d3)>
+#map2 = affine_map<(d0, d1, d2, d3, d4, d5, d6) -> (d0, d1, d4, d5)>
+#map3 = affine_map<(d0, d1, d2, d3) -> (d0, d1, d2, d3)>
+#map4 = affine_map<(d0, d1, d2, d3) -> (d1, d3)>
+
+// This test check that destination-passing-style is respected by fusion.
+// %arg3 should not be used in the scf.forall region.
```

Comment:
You should CHECK-NOT for allocs and copies etc which should not be there anymore.